lib/flipper/cloud/configuration.rb

```diff
@@ -88,14 +82,15 @@ def initialize(options = {})
         @adapter_block = ->(adapter) { adapter }
         self.url = options.fetch(:url) { ENV.fetch("FLIPPER_CLOUD_URL", DEFAULT_URL) }
 
-        instrumenter = options.fetch(:instrumenter, Instrumenters::Noop)
-
         # This is alpha. Don't use this unless you are me. And you are not me.
         cloud_instrument = options.fetch(:cloud_instrument) { ENV["FLIPPER_CLOUD_INSTRUMENT"] == "1" }
-        @instrumenter = if cloud_instrument
-          Instrumenter.new(brow: brow, instrumenter: instrumenter)
-        else
-          instrumenter
+        if cloud_instrument
+          require 'flipper/instrumentation/cloud_subscriber'
+
+          Flipper.instrumenter.subscribe(
+            Flipper::Feature::InstrumentationName,
+            Flipper::Instrumentation::CloudSubscriber.new(brow)
+          )
```

Comment:
Just a heads up that I rewrote this to use a subscriber instead of wrapping the instrumenter. I think it's fair to say that anyone that wants to use this should be using `AS::Notifications` (default in Rails) or some other instrumentation framework.

Doesn't seem like there were any tests for this, and I have not tested it manually yet.

Reply:
The new telemetry stuff needs to work as a wrapper so that and this will conflict.

Reply:
> The main reason is that AS notifications are super slow. I don't want people (including me) to have to use those in order to get analytics.

FWIW, the engine configures AS Notifications by default in all Rails apps, so every app you manage right now is probably already using it.

Reply:
K. I thought we turned that off for some reason but I can see it is not. I'm sure my apps aren't sensitive to it but I've helped a handful of people debug perf issues and AS::Notification was the easiest win each time.